lte/gateway/c/core/oai/tasks/nas/emm/Identification.c

```diff
@@ -53,6 +54,9 @@ extern mme_congestion_params_t mme_congestion_params;
 
 extern int check_plmn_restriction(imsi_t imsi);
 extern int validate_imei(imeisv_t* imeisv);
+extern void create_new_attach_info(
```

Comment:
Would it make sense to move these function definitions to a common header file? Not sure what would be the best place to put these maybe `common_utility_funcs.h` or create a header file under `tasks/nas/emm/` to put these just to avoid having definitions on an implementation file

Reply:
I will move this function to common_utility_funs.cpp where other common functions are defined